packages/app/src/components/inspector/SymbolInstanceOverridesRow.tsx

```diff
@@ -0,0 +1,330 @@
+import { ResetIcon } from '@radix-ui/react-icons';
+import Sketch from '@sketch-hq/sketch-file-format-ts';
+import {
+  InputField,
+  Spacer,
+  TreeView,
+  Select,
+  Button,
+} from 'noya-designsystem';
+import { ApplicationState, Overrides } from 'noya-state';
+import {
+  getSharedStyles,
+  getSharedTextStyles,
+  getSymbols,
+  findSymbolMaster,
+} from 'noya-state/src/selectors/selectors';
+import { memo, ReactNode, useCallback, useMemo } from 'react';
+import { useApplicationState } from '../../contexts/ApplicationStateContext';
+import * as InspectorPrimitives from './InspectorPrimitives';
+interface Props {
+  symbolMaster: Sketch.SymbolMaster;
+  overrideValues: Sketch.OverrideValue[];
+  onSetOverrideValue: (overrideName: string, value: string) => void;
+  onResetOverrideValue: () => void;
+}
+
+type Selectors = {
+  symbols: Sketch.SymbolMaster[];
+  textStyles: Sketch.SharedStyle[];
+  themeStyles: Sketch.SharedStyle[];
+};
+
+const ThemeStyleSelector = ({
+  themeStyles,
+  sharedStyleID,
+  onChange,
+}: {
+  themeStyles: Sketch.SharedStyle[];
+  sharedStyleID: string;
+  onChange: (value: string) => void;
+}) => {
+  const themeStylesOptions = useMemo(
+    () => themeStyles.map((style) => style.do_objectID),
+    [themeStyles],
+  );
+
+  const getThemeStylesTitle = useCallback(
+    (id) => themeStyles.find((style) => style.do_objectID === id)!.name,
+    [themeStyles],
+  );
+
+  return (
+    <Select
+      id="theme-style-selector"
+      value={sharedStyleID}
+      options={themeStylesOptions}
+      getTitle={getThemeStylesTitle}
+      onChange={onChange}
+    />
+  );
+};
+
+const TextStyleSelector = ({
+  textStyles,
+  sharedStyleID,
+  onChange,
+}: {
+  textStyles: Sketch.SharedStyle[];
+  sharedStyleID: string;
+  onChange: (value: string) => void;
+}) => {
+  const textStylesOptions = useMemo(
+    () => textStyles.map((style) => style.do_objectID),
+    [textStyles],
+  );
+
+  const getTextStylesTitle = useCallback(
+    (id) => textStyles.find((style) => style.do_objectID === id)!.name,
+    [textStyles],
+  );
+  return (
+    <Select
+      id="text-style-selector"
+      value={sharedStyleID}
+      options={textStylesOptions}
+      getTitle={getTextStylesTitle}
+      onChange={onChange}
+    />
+  );
+};
+
+const SymbolMasterSelector = ({
+  symbols,
+  symbolId,
+  onChange,
+}: {
+  symbols: Sketch.SymbolMaster[];
+  symbolId: string;
+  onChange: (value: string) => void;
```

Comment:
Should we pass `undefined` in the `"none"` case? Are we assigning `"none"` as the override value currently? (that would work, but probably better to use `undefined`)

Reply:
i believe this has to be a string 🤔

Reply:
I have this as the call for this function. 

```ts
dispatch(
  'setOverrideTextValue',
  overrideName,
  value === '' || value === 'none' ? undefined : value,
);
```